meson.build

```diff
@@ -31,11 +31,12 @@ substs.set('PROJECT_VERSION',      meson.project_version())
 project_source_root = meson.current_source_dir()
 
 want_ossfuzz = get_option('oss-fuzz')
+want_ossfuzz_without_fuzzing_engine = get_option('oss-fuzz-without-fuzzing-engine')
 want_libfuzzer = get_option('llvm-fuzz')
```

Comment:
What is want_libfuzzer for? Is it for building the fuzzers with LF outside of OSS-Fuzz?

Reply:
@jonathanmetzman thank you for taking a look! That's correct. `want_libfuzzer` is used outside of OSS-Fuzz to make it easier to build the fuzzers without `./infra/helper.py` and docker. It's also usually much faster to just install `clang` (and optionally `libfuzzer` depending on which version of clang is installed) and run something like `CC=clang-5.0 CXX=clang++-5.0 ./tools/oss-fuzz.sh`.